internals/utils/capp.go

```diff
@@ -17,8 +17,16 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// AnnotationKeyHasPlacement is the key used to store the managed cluster name in an annotation on a Capp resource.
-const AnnotationKeyHasPlacement = "dana.io/has-placement"
+const (
+	// AnnotationKeyHasPlacement is the key used to store the managed cluster name in an annotation on a Capp resource
+	AnnotationKeyHasPlacement = "dana.io/has-placement"
+	// RcsConfigName is the name of the RCS Deployer Config CRD instance
+	RcsConfigName = "rcs-config"
```

Comment:
couldn't understand what are those for

Reply:
The AnnotationKeyHasPlacement existed before the CR, I simply added it to the other constants. RcsConfigName and RcsConfigNamespace are being used in both the CappPlacement controller and the CappWebhook to retrieve the RCS config object.

Reply:
OK